docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W -T
```

Comment:
Can you also set `-j auto` so we can get some sweet doc build parallelism?

Reply:
Oh Yeah!